impl/src/main/scala/quasar/impl/external/ExternalConfig.scala

```
@@ -46,6 +47,23 @@ object ExternalConfig {
     entriesF.map(ExternalConfig.ExplodedDirs(_))
   }
 
+  def sanitize[F[_]: Sync](cfg: ExternalConfig): F[ExternalConfig] = {
```

The logic in `sanitize` is derived from SDBE, but I'm happy to change it, e.g. `raiseError`  instead of ignoring in l.58